pkg/controllers/controllers.go

```diff
@@ -18,6 +18,7 @@ import (
 	"context"
 
 	"github.com/awslabs/operatorpkg/controller"
+	opevents "github.com/awslabs/operatorpkg/events"
```

Comment:
nit: just use events if there aren't collisions.

Reply:
We do have a collisions https://github.com/aws/karpenter-provider-aws/pull/7382/files#diff-b32dd54d2bd7ecc467b19203287df80c4c1e3934ea80602aadd6658306b5cca6L43